webui/PCWConfig.py

```diff
@@ -87,6 +87,18 @@ def get_providers_for(feature: str, namespace: str):
         return ConfigFile().getList('{}.namespace.{}/providers'.format(feature, namespace),
                                     ConfigFile().getList('{}/providers'.format(feature), ['EC2', 'AZURE', 'GCE']))
 
+    @staticmethod
+    def get_k8s_clusters_for_provider(provider: str) -> list:
+        result = []
+        providers = ConfigFile().get("k8sclusters/providers")
+        if provider in providers:
+            clusters = ConfigFile().get(f"k8sclusters/{provider}-clusters").strip()
```

Comment:
I don't think `strip` is necessary here : 
1. please read carefully `ConfigFile().get` we getting this string from https://docs.python.org/3/library/configparser.html  I hardly believe that they not handling leading/trailing whitespaces in their code 
2. if they really don't ( but I need some proves to believe in that :) ) than we should handle this **within**  ConfigFile().get . Certainly not in every place where we calling `ConfigFile().get`

Reply:
Checked that is not necessary. I tested this and works
`azure-clusters = area1:cluster1, area1:cluster2 `

Reply:
however the next strip is really necessary `cluster = cluster.strip()`

Reply:
done